bindata/network/ovn-kubernetes/common/001-crd.yaml

```diff
@@ -3150,4 +3150,384 @@ spec:
       storage: true
       subresources:
         status: {}
-{{- end }}
\ No newline at end of file
+{{- end }}
+---
```

Comment:
do this only if the feature gate is ON; to be logically correct

Reply:
Done

network-segmentation FG check is added.